libs/ledger-live-common/src/families/aptos/buildTransaction.test.ts

```diff
@@ -0,0 +1,33 @@
+import { createFixtureAccount } from "../../mock/fixtures/cryptoCurrencies";
+import createTransaction from "./createTransaction";
+import buildTransaction from "./buildTransaction";
+import { AptosAPI } from "./api";
+
+jest.mock("./logic", () => ({
+  normalizeTransactionOptions: jest.fn(),
+  DEFAULT_GAS: 100,
+  DEFAULT_GAS_PRICE: 200,
+}));
+
+jest.mock("./api", () => {
+  return {
+    AptosAPI: function () {
+      return {
+        generateTransaction: jest.fn(() => "tx"),
+      };
+    },
+  };
+});
+
+describe("buildTransaction Test", () => {
```

Comment:
We could check that generateTransaction is called with right options, payload, etc

Reply:
added checks